pages/11ty/date.filter.js

```diff
@@ -0,0 +1,8 @@
+module.exports = (config) => {
+  const date = (timestamp) => {
+    const date = new Date(timestamp);
+    return `${date.getDate()}-${date.getMonth() + 1}-${date.getFullYear()}`;
```

Comment:
i don't remember we did discussed time format?)

Reply:
We need to discuss it basically

Reply:
Expected: `26 June 2023`